lib/src/chart/radar_chart/radar_chart.dart

```diff
@@ -18,10 +18,10 @@ class RadarChart extends ImplicitlyAnimatedWidget {
     Duration swapAnimationDuration = const Duration(milliseconds: 150),
     Curve swapAnimationCurve = Curves.linear,
   }) : super(
-      key: key,
-      duration: swapAnimationDuration,
-      curve: swapAnimationCurve,
-  );
+          key: key,
```

Comment:
Please revert this change.

Reply:
it was causing a failure on the pipeline

Reply:
I'm pretty sure that now dev is succeeding in the pipeline.
Please rebase your branch with dev again.
And also please revert InlineSpan to TextSpan

Reply:
ok all set!